SourceKitStressTester/Sources/StressTester/SourceKitDocument.swift

```diff
@@ -280,9 +297,51 @@ struct SourceKitDocument {
     return (info, response)
   }
 
+  private func emitModule() throws {
+    guard let sourceState = sourceState else { return }
+
+    let moduleName = tempModulePath.deletingPathExtension().lastPathComponent
+    let args = self.args.filter { $0 != file.path } + [
```

Comment:
Should we early exit or fail in some way if we don't find file.path in the args?

Reply:
Or maybe we could do that earlier in argument validation?

Reply:
Yeah, can do it right at the start 👍